### ISSUE TITLE
Change link for the status text in the box

### ISSUE THREAD

**Issue by superfigge** [jan-molak/jenkins-build-monitor-plugin#341]
Hi, thanks for a great plugin. We wonder, instead of ending up in the Jenkins job page when clicking jobname status in a box, we want to show another page on the Jenkins machine. As it is now, we first have to click in the box (your nice plugin) and then again on a link. That link shows a deeper result of the job with test results.
So we just want to skip one click and go directly to that result page.
When clicking in the box we for example end up at job/01212%20status/, but we want to directly end up here: 01212%20status/lastSuccessfulBuild/artifact/connTest.html.
In other words, just adding this: lastSuccessfulBuild/artifact/connTest.html to the link.
Can you please give us a little support how we can make this simple change?

Thank you in advance
Br
Tomas Larsson
